### ISSUE TITLE
Improper error handling causing crash when seyren receives a non-json formatted response from graphite

### ISSUE THREAD

**Issue by isavoy** [scobal/seyren#342]
Sometimes graphite returns a non-json response - for instance, a 500 response from django for whatever reason. When seyren happens to poll graphite and it gets an invalid response, seyren generates a stack trace, and will fail on every subsequent attempt to gather data from graphite - regardless of whether a 500 error is still being generated or not.

Also, when I say "fail on every subsequent attempt to gather data from graphite", I mean it fails on every metric, not just the one that triggered the 500 response. Seyren never recovers without intervention.

We are using the 1.3.0 release.

Here is an example of one of the tracebacks:
```
[seyren.check-scheduler-4] WARN   c.s.c.s.c.GraphiteTargetChecker - any_metric now and 1w ago as % failed to read from Graphite
 com.seyren.core.util.graphite.GraphiteReadException: Failed to read from Graphite
	at com.seyren.core.util.graphite.GraphiteHttpClient.getTargetJson(GraphiteHttpClient.java:136) ~[seyren-core-1.3.0.jar:na]
	at com.seyren.core.service.checker.GraphiteTargetChecker.check(GraphiteTargetChecker.java:50) ~[seyren-core-1.3.0.jar:na]
	at com.seyren.core.service.schedule.CheckRunner.run(CheckRunner.java:65) ~[seyren-core-1.3.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471) ~[na:1.6.0_27]
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334) ~[na:1.6.0_27]
	at java.util.concurrent.FutureTask.run(FutureTask.java:166) ~[na:1.6.0_27]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$101(ScheduledThreadPoolExecutor.java:165) ~[na:1.6.0_27]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:266) ~[na:1.6.0_27]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146) ~[na:1.6.0_27]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) ~[na:1.6.0_27]
	at java.lang.Thread.run(Thread.java:701) ~[na:1.6.0_27]
Caused by: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('<' (code 60)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')
 at [Source: org.apache.http.conn.EofSensorInputStream@1e292edb; line: 1, column: 2]
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1486) ~[jackson-core-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:518) ~[jackson-core-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:447) ~[jackson-core-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._handleUnexpectedValue(UTF8StreamJsonParser.java:2486) ~[jackson-core-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._nextTokenNotInObject(UTF8StreamJsonParser.java:801) ~[jackson-core-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser.nextToken(UTF8StreamJsonParser.java:697) ~[jackson-core-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:3600) ~[jackson-databind-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3545) ~[jackson-databind-2.5.0.jar:2.5.0]
	at com.fasterxml.jackson.databind.ObjectMapper.readTree(ObjectMapper.java:2074) ~[jackson-databind-2.5.0.jar:2.5.0]
	at com.seyren.core.service.checker.JsonNodeResponseHandler.handleResponse(JsonNodeResponseHandler.java:37) ~[seyren-core-1.3.0.jar:na]
	at com.seyren.core.service.checker.JsonNodeResponseHandler.handleResponse(JsonNodeResponseHandler.java:28) ~[seyren-core-1.3.0.jar:na]
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:218) ~[httpclient-4.3.6.jar:4.3.6]
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:160) ~[httpclient-4.3.6.jar:4.3.6]
	at com.seyren.core.util.graphite.GraphiteHttpClient.getTargetJson(GraphiteHttpClient.java:134) ~[seyren-core-1.3.0.jar:na]
	... 10 common frames omitted
```

**Comment by superyizhi** [comment]
i have this problem too

**Comment by scobal** [comment]
This will be solved as part of #256 I think
